File: C++/media.cpp
#include <stdio.h>
#include <locale.h>

int main()
{
	setlocale(LC_ALL,"Portuguese");
	float nota1, nota2, media;
	printf("Digite sua primeira nota: ");
	scanf("%f", &nota1);
	printf("Digite sua segunda nota: ");
	scanf("%f", &nota2);
	media=(nota1+nota2)/2;
	if(media >= 7) {
		printf("Você foi aprovado! Com média: %f", media);
	} else {
		printf("Você foi reprovado! Com média: %f", media);
	}
	return 0;
}

File: Python/media.py
nota1 = input("Digite sua primeira nota: ")
nota2 = input("Digite sua segunda nota: ")

if((float(nota1)+float(nota2))/2 >= 7):
    print("Sua media: ", (float(nota1)+float(nota2))/2)
    print("Você esta aprovado!")
else:
    print("Sua media: ", (float(nota1)+float(nota2))/2)
    print("Você está reprovado!")
